lib/rust/parser/tests/parse.rs

```diff
@@ -361,6 +361,55 @@ fn plus_negative() {
     test(&code.join("\n"), expected);
 }
 
+#[test]
+fn minus_binary() {
+    let cases = [
+        ("x - 5", block![(OprApp (Ident x) (Ok "-") (Number 5))]),
+        ("x-5", block![(OprApp (Ident x) (Ok "-") (Number 5))]),
+        ("x.-y", block![(OprApp (Ident x) (Ok ".") (UnaryOprApp "-" (Ident y)))]),
```

Comment:
@jdunkerley mentioned to me that he'd like to see `-7.absolute` to evaluate to `7` and not `-7` as is currently the case...